tsconfig.json

```diff
@@ -2,6 +2,7 @@
   "extends": "./node_modules/gts/tsconfig-google.json",
   "compilerOptions": {
     "lib": ["es2018", "dom"],
+    "skipLibCheck": true,
```

Comment:
Will create a follow-up issue to revert this in the next major (should be soon).

Reply:
Background: https://www.typescriptlang.org/tsconfig#skipLibCheck

There's a [broken type issue](https://github.com/googleapis/gcp-metadata/actions/runs/5267177572/jobs/9522094784?pr=561) that would require a major version bump to resolve.

Reply:
I too got bitten by this broken type issue

Reply:
Tracking issue: https://github.com/googleapis/gcp-metadata/issues/564